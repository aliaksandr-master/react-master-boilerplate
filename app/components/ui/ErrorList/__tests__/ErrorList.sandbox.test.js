/* eslint-env browser, jest */
/* eslint-disable react/jsx-no-bind */

import React from 'react';
import renderer from 'react-test-renderer';
import { wrapSandbox } from '../../../../vendor/react-sandbox/sandbox-staging';
import sandboxErrorList from '../ErrorList.sandbox';
import Test from '../../../test/Test';



describe('rendering', () => {
  wrapSandbox(sandboxErrorList).forEach(({ name, template, props }) => {
    it(`should render sandbox "${name}"`, () => {
      const ErrorListSandboxComponent = template();

      const component = renderer.create(
        <Test>
          <ErrorListSandboxComponent {...props} />
        </Test>
      );

      expect(component.toJSON()).toMatchSnapshot();
    });
  });
});
