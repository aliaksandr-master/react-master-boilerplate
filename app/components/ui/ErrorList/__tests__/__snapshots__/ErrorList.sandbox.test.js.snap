// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering should render sandbox "default" 1`] = `
<div>
  <div
    className="ErrorList ErrorList--themeDefault ErrorList--sizeDefault"
  >
    <div
      className="ErrorListValidation"
    >
      Hello World : First Error!
    </div>
    <div
      className="ErrorListValidation"
    >
      Foo : Error 2!
    </div>
  </div>
</div>
`;
